subprojects/internal-testing/src/main/groovy/org/gradle/test/preconditions/UnitTestPreconditions.groovy

```diff
@@ -22,51 +22,48 @@ import org.gradle.internal.os.OperatingSystem
 import org.gradle.test.precondition.TestPrecondition
 import org.testcontainers.DockerClientFactory
 
-import static org.gradle.test.precondition.TestPrecondition.doSatisfies;
```

Comment:
The static import is missing and that is why compilation fails.

Reply:
For some reason, IntelliJ believes that these static imports are not needed...